shared/src/business/utilities/htmlGenerator/index.scss

```diff
@@ -560,6 +560,18 @@ th {
   }
 }
 
+.notice-of-trial-title {
+  margin-bottom: 0px;
+}
+
+#notice-of-trial-pdf .info-box {
+  margin-bottom: 20px;
+}
+
+#notice-body > p {
+  margin-top: 0px;
```

Comment:
_nit_: zero-values in CSS require no units (i.e. can be written as `margin-bottom: 0`).  I thought we had a linter rule that complains about this.  No biggie.